src/transformers/models/omdet_turbo/processing_omdet_turbo.py

```diff
@@ -290,66 +335,74 @@ def post_process_grounded_object_detection(
         Args:
             outputs ([`OmDetTurboObjectDetectionOutput`]):
                 Raw outputs of the model.
-            classes (Union[List[str], List[List[str]]]): The input classes names.
-            score_threshold (float, defaults to 0.3): Only return detections with a confidence score exceeding this
-                threshold.
-            nms_threshold (float, defaults to 0.5):  The threshold to use for box non-maximum suppression. Value in [0, 1].
-            target_sizes (`torch.Tensor` or `List[Tuple[int, int]]`, *optional*, defaults to None):
+            text_labels (Union[List[str], List[List[str]]], *optional*):
+                The input classes names. If not provided, classes will be set to `None` in `outputs`.
```

Comment:
```suggestion
                The input classes names. If not provided, `text_labels` will be set to `None` in `outputs`.
```
Unless I misunderstood.

Reply:
You are right 🤗